setup.cfg

```diff
@@ -25,7 +25,7 @@ install_requires =
     pyyaml >= 5.3
     click == 7.1.2
     requests == 2.24
-    opencv-python == 4.1.2.30
+    opencv-contrib-python == 4.1.2.30
```

Comment:
Is it possible to merge `dev` into `node-v1.2` and then I'll sync my branch with `node-v1.2` to avoid potential conflicts later? And also, `opencv-python==4.5.2.54` required some changes to draw nodes code which are not present on `node-v1.2` which this branch is based on.

Reply:
Synced, you can go ahead with these changes.

Reply:
Updated requirements, made to necessary changes to use mosse through cv2 legacy api as well.